minvws/nl-kat-coordination
# Replace finding description 'None' with the id

### Changes
When a finding is pending and doesn't have a description yet, the **aggregate report** showed "None - Pending" in the table.
This wasn't clear for the user, so this PR changes "None" to the finding ID.

This PR also fixes the host that wasn't visible. 

### Issue link
Closes #3759

### Demo
![afbeelding](https://github.com/user-attachments/assets/3f2e5b48-c89f-4766-af46-ffc2d08db574)


### QA notes
Create an Aggregate Report.
Go to chapter "System specific" and check the compliance table

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
